src/test/scala/org/broadinstitute/dsde/firecloud/metrics/MetricsSpec.scala

```diff
@@ -87,3 +127,16 @@ class MetricsSpecMockRawlsDAO(numFailures: Int, numSamples: Int) extends MockRaw
     }
   }
 }
+
+class MetricsSpecMockSearchDAO(numFailures: Int, numSubjects: Int) extends MockSearchDAO {
```

Comment:
Could you add some comments about the intended behavior of these MetricsSpecMocks?  It's not obvious to me what `numFailures` is doing.